tests/test_app_runner.py

```diff
@@ -12,6 +11,15 @@
 from soar_sdk.webhooks.models import WebhookRequest, WebhookResponse
 
 
+@pytest.fixture
+def tmp_asset_and_param_files(tmp_path: Path) -> tuple[Path, Path]:
+    """Fixture to create temporary asset and parameter files, already opened for writing."""
+    asset_file = tmp_path / "asset.json"
+    param_file = tmp_path / "params.json"
+
+    return asset_file.open("w+"), param_file.open("w+")
```

Comment:
Nit: I don't think this return type is correct? `.open()` doesn't return a file. Also, this won't close them? pytest supports generator fixtures which should do the trick:
```suggestion
def tmp_asset_and_param_files(tmp_path: Path) -> Iterator[tuple[TextIO, TextIO]]:
    """Fixture to create temporary asset and parameter files, already opened for writing."""
    asset_file = tmp_path / "asset.json"
    param_file = tmp_path / "params.json"

    with asset_file.open("w+") as af, param_file.open("w+") as pf:
        yield af, pf
```

Reply:
you're right, fixed